common/common-rest/src/main/java/io/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -216,7 +216,7 @@ protected void sendResponse(Response response) throws Exception {
       }
     }
     responseEx.setStatus(response.getStatusCode(), response.getReasonPhrase());
-    responseEx.setContentType(produceProcessor.getName());
+    responseEx.setContentType(produceProcessor.getName()+"; charset=utf-8");
```

Comment:
It's better to get the charset from context or other setting to let the user override it.

Reply:
yes
but too many place need to be changed, so it will be a new issue, not in this PR.